app/src/main/java/com/sun_asterisk/myeditor03/screen/collectiondetail/CollectionsDetailFragment.kt

```diff
@@ -1,14 +1,42 @@
 package com.sun_asterisk.myeditor03.screen.collectiondetail
 
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
 import android.os.Bundle
 import android.support.v4.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
+import android.view.View.OnClickListener
 import android.view.ViewGroup
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
 import com.sun_asterisk.myeditor03.R
+import com.sun_asterisk.myeditor03.data.model.Collection
+import com.sun_asterisk.myeditor03.data.model.Photo
+import com.sun_asterisk.myeditor03.data.source.PhotoRepository
+import com.sun_asterisk.myeditor03.data.source.local.PhotoLocalDataSource
+import com.sun_asterisk.myeditor03.data.source.remote.PhotoRemoteDataSource
+import com.sun_asterisk.myeditor03.screen.collectiondetail.adapter.CollectionDetailAdapter
+import com.sun_asterisk.myeditor03.screen.collections.CollectionsFragment
+import com.sun_asterisk.myeditor03.utils.CommonUtils
+import com.sun_asterisk.myeditor03.utils.EndlessScrollListener
+import com.sun_asterisk.myeditor03.utils.MyViewModelFactory
 import com.sun_asterisk.myeditor03.utils.OnItemRecyclerViewClickListener
+import com.sun_asterisk.myeditor03.utils.removeFragment
+import kotlinx.android.synthetic.main.fragment_collection_detail.imageBack
+import kotlinx.android.synthetic.main.fragment_collection_detail.imageViewPoster
+import kotlinx.android.synthetic.main.fragment_collection_detail.recyclerPhotoByCollection
+import kotlinx.android.synthetic.main.fragment_collection_detail.textViewTitle
+import kotlinx.android.synthetic.main.fragment_collection_detail.textViewTotalCount
 
-class CollectionsDetailFragment : Fragment(), OnItemRecyclerViewClickListener<String> {
+class CollectionsDetailFragment : Fragment(), OnItemRecyclerViewClickListener<Photo>, OnClickListener {
+    private val TARGET = "h=128&w=128"
+    private val REPLACEMENT = "h=256&w=256"
+    private lateinit var collectionDetailAdapter: CollectionDetailAdapter
```

Comment:
change to local

Reply:
adapter còn dùng để hứng data dưới livedata nên ko khái báo cục bộ đc ạ